react-debugging.html

```diff
@@ -26,9 +24,10 @@
           setMood: null
         }
       }
-
       nextMood = () => {
-        if(currentMood === moods.length - 1){
+        let currentMood = 0
+        //was getting an error saying that currentMood was undefined so I declared it above, then it said moods was undefined. so I added this.state, I got it to work but it only works for one click. Afterwards clicking it again doesn't do anything and it doesn't give me errors on the browser console. At this point I'm kind of stuck.
+        if(this.currentMood === this.state.moods.length - 1){
```

Comment:
And this is where current mood is continuing to fail, it needs to be this.state.currentMood to reference to the state, not the variable you created.